web/ui/dashboard/src/app/private/components/create-project-component/create-project-component.component.ts

```diff
@@ -272,14 +278,14 @@ export class CreateProjectComponent implements OnInit {
 	}
 
 	cancel() {
-		this.confirmModal = true;
+		this.confirmationDialog.nativeElement.showModal()
```

Comment:
Thinking, is this modal really useful or it's just a hindrance?

The discard button that calls this function does not have a high tendency to be clicked by mistake; besides, there is also a close button at the top of the modal (for scenarios where it is used within a modal).

I think you should take out the discard button entirely, hence the modal here. I'll also update it in the design.